src/NHibernate/NHibernate.csproj

```diff
@@ -1,1819 +1,41 @@
-    <Compile Include="Linq\Visitors\ResultOperatorProcessors\ProcessSkip.cs" />
-    <Compile Include="Linq\Visitors\ResultOperatorProcessors\ProcessTake.cs" />
-    <Compile Include="Linq\Visitors\SwapQuerySourceVisitor.cs" />
-    <Compile Include="Linq\Visitors\EqualityHqlGenerator.cs" />
-    <Compile Include="Linq\Visitors\ExpressionParameterVisitor.cs" />
-    <Compile Include="Linq\NhLinqExpression.cs" />
-    <Compile Include="Linq\NhLinqExpressionReturnType.cs" />
-    <Compile Include="Linq\DefaultQueryProvider.cs" />
-    <Compile Include="Linq\Visitors\ExpressionKeyVisitor.cs" />
-    <Compile Include="Linq\GroupBy\IsNonAggregatingGroupByDetectionVisitor.cs" />
-    <Compile Include="Linq\GroupBy\AggregatingGroupByRewriter.cs" />
-    <Compile Include="Linq\GroupJoin\AggregatingGroupJoinRewriter.cs" />
-    <Compile Include="Linq\ResultOperators\ClientSideTransformOperator.cs" />
-    <Compile Include="Linq\ExpressionToHqlTranslationResults.cs" />
-    <Compile Include="Linq\EnumerableHelper.cs" />
-    <Compile Include="Linq\GroupBy\GroupBySelectClauseRewriter.cs" />
-    <Compile Include="Linq\Visitors\HqlGeneratorExpressionVisitor.cs" />
-    <Compile Include="Linq\LinqExtensionMethods.cs" />
-    <Compile Include="Linq\ReWriters\MergeAggregatingResultsRewriter.cs" />
-    <Compile Include="Param\NamedParameter.cs" />
-    <Compile Include="Linq\Visitors\NhExpressionVisitor.cs" />
-    <Compile Include="Linq\Expressions\NhNewExpression.cs" />
-    <Compile Include="Linq\NhQueryable.cs" />
-    <Compile Include="Linq\ResultOperators\NonAggregatingGroupBy.cs" />
-    <Compile Include="Linq\GroupBy\NonAggregatingGroupByRewriter.cs" />
-    <Compile Include="Linq\Visitors\SelectClauseVisitor.cs" />
-    <Compile Include="Linq\Visitors\QueryModelVisitor.cs" />
-    <Compile Include="Linq\ResultTransformer.cs" />
-    <Compile Include="Linq\ReWriters\QueryReferenceExpressionFlattener.cs" />
-    <Compile Include="Linq\Visitors\VisitorParameters.cs" />
-    <Compile Include="Loader\Criteria\ComponentCollectionCriteriaInfoProvider.cs" />
-    <Compile Include="Loader\Criteria\EntityCriteriaInfoProvider.cs" />
-    <Compile Include="Loader\Criteria\ICriteriaInfoProvider.cs" />
-    <Compile Include="Loader\Criteria\ScalarCollectionCriteriaInfoProvider.cs" />
-    <Compile Include="Logging.cs" />
-    <Compile Include="Mapping\ISqlCustomizable.cs" />
-    <Compile Include="Param\AbstractExplicitParameterSpecification.cs" />
-    <Compile Include="Param\AggregatedIndexCollectionSelectorParameterSpecifications.cs" />
-    <Compile Include="Param\CollectionFilterKeyParameterSpecification.cs" />
-    <Compile Include="Param\DynamicFilterParameterSpecification.cs" />
-    <Compile Include="Param\IExplicitParameterSpecification.cs" />
-    <Compile Include="Param\IParameterSpecification.cs" />
-    <Compile Include="Param\NamedParameterSpecification.cs" />
-    <Compile Include="Para
```

Comment:
System.Reflection.Emit is available only for netstandard1.0 + platform extensions (whatever it does mean)